feat: explain polkadotjs (#239)
KILTprotocol/docs: docs/concepts/09_glossary.md

A multi-chain network that allows for interoperability between different blockchain protocols, including the KILT Protocol.
More details about Polkadot can be found on their [offical documentation](https://wiki.polkadot.network/docs/getting-started#what-is-polkadot)

### polkadot.js

> A JavaScript library that allows developers to interact with the substrate based blockchains, including the KILT Protocol.

Reviewer: ```suggestion
A JavaScript library that allows developers to interact with Substrate-based blockchains, including KILT Spiritnet and Peregrine.
```

> It is the basis of the KILT SDK and provides many utils and functions that may be usefull for application developers.

Reviewer: ```suggestion
It is the basis of the KILT SDK and provides many utils and functions that may be useful for application developers.
```
